packages/lesswrong/server/callbacks/userCallbacks.tsx

```diff
@@ -464,6 +464,14 @@ getCollectionHooks("Users").updateBefore.add(async function UpdateDisplayName(da
       throw new Error("This display name is already taken");
     }
   }
+  if (data.shortformFeedId) {
+    void updateMutator({
+      collection: Posts,
+      documentId: data.shortformFeedId,
+      set: {title: userShortformPostTitle(data)},
+      validate: false,
+    });
+  }
```

Comment:
```suggestion
      throw new Error("This display name is already taken");
    }
    if (data.shortformFeedId) {
      void updateMutator({
        collection: Posts,
        documentId: data.shortformFeedId,
        set: {title: userShortformPostTitle(data)},
        validate: false,
      });
    }
  }
```
I think this should run inside the block that checks if the display name has actually changed, so as not to make unnecessary updates when the user edits anything on their profile.

Reply:
Agreed.